test/test_pytest_fixtures.py

```diff
@@ -34,3 +36,8 @@ def test_can_save_model(db_session):
 
 def test_db_is_empty(db_session):
     assert not db_session.query(User).all()
+
+
+def test_needs_to_override_model_base():
```

Comment:
Might be nicer to test this using the `testdir` fixture, because directly importing short-circuits pytest's discovery process.

See https://github.com/onefinestay/nameko/blob/master/test/testing/test_pytest_plugin.py for examples